### PR TITLE
Ggplot2coord equal

### LINKED ISSUE #5214: Addition to Plot Options - coordinates tab
There are a number of useful coordinates functions that could be added to this tab. One I find very useful is
`ggplot2::coord_equal` which makes x and y the same scale. This one would be easy to add as a simple check box.

When someone does this, can you also look at the other functions, there are quite a few that start with `ggplot2::coord_`. Eventually we should have them all. I would like `coord_equal` added first because its simple, but if any others look very useful they could also be added at the same time.

### PR DESCRIPTION
**Pull request by Ivanluv**
@Muthenya please review this

Fixes #5214

### CODE REVIEW

**Comment by maxwellfundi** [comment]
reviewing this now.

File: instat/sdgPlots.vb [hunk 1]
```diff
@@ -213,13 +216,23 @@ Public Class sdgPlots
...
+
+
+        UcrChkSameScale.SetText("Same Scale (coord-equal)")
+        clsCoordEqualFunc.SetPackageName("ggplot2")
+        clsCoordEqualFunc.SetRCommand("coord_equal")
+        clsCoordEqualParam.SetArgumentName("coord_equal")
+        clsCoordEqualFunc.AddParameter("ratio", "1", iPosition:=1)
+        clsCoordEqualParam.SetArgument(clsCoordEqualFunc)
+        UcrChkSameScale.SetParameter(clsCoordEqualParam, bNewChangeParameterValue:=False, bNewAddRemoveParameter:=True)
+        UcrChkSameScale.AddParameterPresentCondition(True, "coord_equal", True)
```
Comment (maxwellfundi) [inline instat/sdgPlots.vb]: @Ivanluv  The naming convention here is not correct. We start the names of controls with lowercase

Comment (maxwellfundi) [inline instat/sdgPlots.vb]:
```suggestion
        ucrChkSameScale.AddParameterPresentCondition(True, "coord_equal", True)
```

File: instat/sdgPlots.vb [hunk 2]
```diff
@@ -361,7 +374,8 @@ Public Class sdgPlots
         'Themes tab
         SetRcodeForCommonThemesControls(bReset)
         'coordinates tab
-        ucrChkHorizontalplot.SetRCode(clsBaseOperator, bReset, bCloneIfNeeded:=True)
+        ucrChkHorizontalPlot.SetRCode(clsBaseOperator, bReset, bCloneIfNeeded:=True)
+        UcrChkSameScale.SetRCode(clsBaseOperator, bReset, bCloneIfNeeded:=True)
```
Comment (maxwellfundi) [inline instat/sdgPlots.vb]: @Ivanluv  Please change the case for the `UcrChkSameScale` to `ucrChkSameScale`  and this will be good to go in.

**Comment by maxwellfundi** [comment]
@dannyparsons  this is ready on changing the case for  the new checkbox.

**Comment by maxwellfundi** [comment]
Nice that you have made the change @Ivanluv . This is now ready @dannyparsons

File: instat/sdgPlots.vb [hunk 3]
```diff
@@ -213,13 +216,23 @@ Public Class sdgPlots
...
-        ucrChkHorizontalplot.AddParameterPresentCondition(True, "coord_flip", True)
+        ucrChkHorizontalPlot.SetParameter(clsCoordFlipParam, bNewChangeParameterValue:=False, bNewAddRemoveParameter:=True)
+        ucrChkHorizontalPlot.AddParameterPresentCondition(True, "coord_flip", True)
+
+
+        ucrChkSameScale.SetText("Same Scale (coord-equal)")
+        clsCoordEqualFunc.SetPackageName("ggplot2")
+        clsCoordEqualFunc.SetRCommand("coord_equal")
+        clsCoordEqualParam.SetArgumentName("coord_equal")
+        clsCoordEqualFunc.AddParameter("ratio", "1", iPosition:=1)
```
Comment (dannyparsons) [inline instat/sdgPlots.vb]: Since `1` is the default, you don't need to add it in the function.

**Comment by dannyparsons** [comment]
Looks good just one minor change.

**Comment by maxwellfundi** [comment]
@dannyparsons the small change here is done. ready for you.

**Pull request merged**